--- libsoup-3.6.5/meson.build.orig	2025-03-21 19:30:16.000000000 +0100
+++ libsoup-3.6.5/meson.build	2025-04-04 18:11:50.227681858 +0200
@@ -325,11 +325,18 @@ endif
 ################
 # NTLM not supported on Windows
 if host_machine.system() != 'windows'
-  ntlm_auth = find_program(get_option('ntlm_auth'), required: get_option('ntlm'))
+  ntlm_auth_path = get_option('ntlm_auth')
+  if ntlm_auth_path[0] == '/'
+    ntlm_auth_found = true
+  else
+    ntlm_auth = find_program(get_option('ntlm_auth'), required: get_option('ntlm'))
+    ntlm_auth_found = ntlm_auth.found()
+    ntlm_auth_path = ntlm_auth.path()
+  endif
 
-  if ntlm_auth.found()
+  if ntlm_auth_found
     add_project_arguments('-DUSE_NTLM_AUTH=1', language : 'c')
-    add_project_arguments('-DNTLM_AUTH="' + ntlm_auth.path() + '"', language : 'c')
+    add_project_arguments('-DNTLM_AUTH="' + ntlm_auth_path + '"', language : 'c')
   endif
 else
   ntlm_auth = dependency('', required: false)
@@ -388,7 +395,7 @@ summary({
 
 summary({
     'GSSAPI' : gssapi_dep.found(),
-    'NTLM' : ntlm_auth.found(),
+    'NTLM' : ntlm_auth_found,
     'Brotli' : brotlidec_dep.found(),
     'Translations' : xgettext.found(),
     'GIR' : enable_introspection,
