--- libsoup-3.6.6/meson.build.orig	2026-02-14 23:13:32.000000000 +0100
+++ libsoup-3.6.6/meson.build	2026-02-20 18:46:46.063702246 +0100
@@ -326,11 +326,18 @@ endif
 ################
 # NTLM not supported on Windows
 if host_machine.system() != 'windows'
-  ntlm_auth = find_program(get_option('ntlm_auth'), required: get_option('ntlm'))
+  ntlm_auth_path = get_option('ntlm_auth')
+  if ntlm_auth_path[0] == '/'
+    ntlm_auth_found = true
+  else
+    ntlm_auth = find_program(get_option('ntlm_auth'), required: get_option('ntlm'))
+    ntlm_auth_found = ntlm_auth.found()
+    ntlm_auth_path = ntlm_auth.path()
+  endif
 
-  if ntlm_auth.found()
+  if ntlm_auth_found
     add_project_arguments('-DUSE_NTLM_AUTH=1', language : 'c')
-    add_project_arguments('-DNTLM_AUTH="' + ntlm_auth.full_path() + '"', language : 'c')
+    add_project_arguments('-DNTLM_AUTH="' + ntlm_auth_path + '"', language : 'c')
   endif
 else
   ntlm_auth = dependency('', required: false)
@@ -393,7 +400,7 @@ summary({
 
 summary({
     'GSSAPI' : gssapi_dep.found(),
-    'NTLM' : ntlm_auth.found(),
+    'NTLM' : ntlm_auth_found,
     'Brotli' : brotlidec_dep.found(),
     'Translations' : xgettext.found(),
     'GIR' : enable_introspection,
