--- libsoup-2.99.3/meson.build.orig	2021-04-08 21:34:56.261188700 +0200
+++ libsoup-2.99.3/meson.build	2021-04-11 15:13:46.597820547 +0200
@@ -329,11 +329,18 @@ endif
 ################
 # NTLM not supported on Windows
 if host_machine.system() != 'windows'
-  ntlm_auth = find_program(get_option('ntlm_auth'), required: get_option('ntlm'))
+  ntlm_auth_path = get_option('ntlm_auth')
+  if ntlm_auth_path[0] == '/'
+    ntlm_auth_found = true
+  else
+    ntlm_auth = find_program(get_option('ntlm_auth'), required: get_option('ntlm'))
+    ntlm_auth_found = ntlm_auth.found()
+    ntlm_auth_path = ntlm_auth.path()
+  endif
 
-  if ntlm_auth.found()
+  if ntlm_auth_found
     add_project_arguments('-DUSE_NTLM_AUTH=1', language : 'c')
-    add_project_arguments('-DNTLM_AUTH="' + ntlm_auth.path() + '"', language : 'c')
+    add_project_arguments('-DNTLM_AUTH="' + ntlm_auth_path + '"', language : 'c')
   endif
 endif
 
@@ -392,7 +399,7 @@ summary({
 
 summary({
     'GSSAPI' : enable_gssapi,
-    'NTLM' : ntlm_auth.found(),
+    'NTLM' : ntlm_auth_found,
     'Brotli' : brotlidec_dep.found(),
     'Translations' : xgettext.found(),
     'GIR' : enable_introspection,
